targets/zephyr/Makefile.zephyr

```diff
@@ -47,6 +47,11 @@ endif
 
 INTERM     = build/$(BOARD)/obj-$(BOARD)
 OUTPUT     = build/$(BOARD)
+DOTCONFIG  = $(OUTPUT)/zephyr/.config
+
+include $(DOTCONFIG)
```

Comment:
If I get it right, then `.config` is included unconditionally, and it seems that `ARCH` comes from that file, and then `CPU` is defined based on `ARCH`. Thus, if the file does not exist yet, then almost none of the "build" recipes can be invoked since the evaluation of `CPU` would trigger an error. So, `make -f Makefile.zephyr .config` (or `make -f Makefile.zephyr initconfig`, as suggested in this review) seems to be a mandatory first step in the build process. If so, then the readme should be updated as well.